lmdeploy/pytorch/backends/cuda/attention.py

```diff
@@ -210,6 +238,7 @@ def build(
         sliding_window: int = None,
         logical_softcapping: float = None,
         causal: bool = True,
+        use_flash_mla: bool = False,
```

Comment:
If you change the API, `backends/attention.py` and all other backends(dinfer) should also be updated.